src/main.ts

```diff
@@ -290,15 +290,14 @@ export default class SRPlugin extends Plugin {
             const tags = getAllTags(fileCachedData) || [];
 
             let shouldIgnore = true;
-            for (const tag of tags) {
-                if (
-                    this.data.settings.tagsToReview.some(
-                        (tagToReview) => tag === tagToReview || tag.startsWith(tagToReview + "/")
-                    )
-                ) {
-                    if (!Object.prototype.hasOwnProperty.call(this.reviewDecks, tag)) {
-                        this.reviewDecks[tag] = new ReviewDeck(tag);
+            const matchedNoteTags = [];

```

Comment:
added this to store the matched tags otherwise I would had to repeat the logic later in the function.